tests/integration/test_charm.py

```diff
@@ -56,6 +69,55 @@ async def test_build_and_deploy(ops_test: OpsTest):
     assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
 
 
+@pytest.mark.abort_on_fail
+async def test_configmap_created(lightkube_client: Client, ops_test: OpsTest):
```

Comment:
I think this test case can live in unit test, but if it is too much of a problem, I'm okay with it.